extensions/gitpod-web/src/extension.ts

```diff
@@ -497,6 +499,10 @@ class GitpodPortViewProvider implements vscode.WebviewViewProvider {
 			if (!port) { return; }
 			if (message.command === 'urlCopy' && port.status.exposed) {
 				await vscode.env.clipboard.writeText(port.status.exposed.url);
+				this.context.fireAnalyticsEvent({
+					eventName: 'vscode_execute_command_gitpod_ports',
+					properties: { action: 'urlCopy', isWebview: true, userOverride: isUserOverrideSetting('gitpod.experimental.portsView.enabled') ? 'on' : 'off' }
```

Comment:
Maybe just use `String(isUserOverrideSetting('gitpod.experimental.portsView.enabled'))` so it converts to common value `'true'` `'false'` instead of mapping to custom values